src/portal/lib/src/service/http-xsrf-token-extractor.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+import { HttpXsrfTokenExtractor } from "@angular/common/http";
+import { CookieService } from "ngx-cookie";
+@Injectable()
+export class HttpXsrfTokenExtractorDefault extends HttpXsrfTokenExtractor {
```

Comment:
Let's rename it like `BeegoXsrfTokenExtractor` because it's not the `default` one....